commands/orgs.js

```diff
@@ -123,8 +123,16 @@ async function orgsCli (details, org) {
 function printHelp () {
   helpHeader(
     'orgs',
-    chalk`ncm {${COLORS.yellow} orgs} {${COLORS.teal} [<orgname>] [options]}`,
-    'ncm orgs [<orgname>] [options]'
+    chalk`{${COLORS.light1} ncm} {${COLORS.yellow} orgs} {${COLORS.teal} [<orgname>] [options]}`,
+    'ncm orgs [<orgname>] [options]',
+    chalk`
+Change your active NodeSource organization, which only impacts the whitelist.
```

Comment:
```suggestion
Change your active NodeSource organization, which impacts the whitelist.
```

No need to undersell this part.